modules/ROOT/partials/configuration/powerpaste_clean_filtered_inline_elements.adoc

```diff
@@ -5,7 +5,9 @@ This option allows for configuration of PowerPaste's *"clean"* paste filters for
 
 The list of inline elements that should be removed on paste can be specified by setting `+powerpaste_clean_filtered_inline_elements+` to a comma-separated string of inline element tag names.
 
-*Possible values:* A comma-separated string.
+*Type:* A comma-separated string.
+
+*Default value:* `+[]+`
```

Comment:
Should this be an empty string instead of an array?

Reply:
Probably, though it should just not be set if it's a string... I didn't check that, I just copied what was mentioned in the comments. I'll double check I didn't add it to the wrong thing.

Reply:
Ahh it's because this is a `string[]` option type so it allows both a comma separated string or an array of strings. I'll fix the `Type` instead. Thanks for the catch!

Reply:
Done